RELEASING.md

```diff
@@ -0,0 +1,12 @@
+# Releasing
+
+Note: There is not yet a formal release process for this repository, however there is a `STABLE_PROTOS_HASH` variable in the [Makefile](Makefile) which is used for breaking change detection, and needs to be updated to match the commit used by Fabric.
```

Comment:
Ok good, moving in the right direction.
Can you describe what exactly breaking change detection is and why we care? Is the intent to avoid any breaking change? If so, why not check every PR against the latest so that we never have a break and eliminate the need to maintain a STABLE_PROTOS_HASH? Otherwise there could be latent breaks that get committed for months, but not identified until STABLE_PROTOS_HASH is updated. Clearly I'm still not completely following the whole intent :-)

Who is responsible for updating STABLE_PROTOS_HASH? Somebody contributing to this repo? Or the maintainer that updates the fabric fabric-protos-go dependency? If the latter then we should also put a comment in the fabric go.mod next to the fabric-protos-go dependency. I suspect people will forget to update STABLE_PROTOS_HASH, I'd prefer to not add it at all, but if it is required then we need to be very clear about its update lifecycle.

Finally, it is not a commit from this repo that Fabric uses, it is a commit from fabric-protos-go that fabric uses, while  STABLE_PROTOS_HASH references the commit in this repo that is used to generate the fabric-protos-go commit. There is plenty of room here for confusion, which is why I'm being picky about providing clear contribution guidance.